easybuild/easyconfigs/t/Trim_Galore/Trim_Galore-0.3.7.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = "PackedBinary"
+
+name = 'Trim_Galore'
+version = '0.3.7'
+
+homepage = 'http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'
+description = """A wrapper tool around Cutadapt and FastQC to consistently apply 
+quality and adapter trimming to FastQ files, with some extra functionality for 
+MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(namelower)s_v%(version)s.zip']
+source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s']
+
+install_cmd = "cp -a %(namelower)s_zip/* %(installdir)s && chmod +x %(installdir)s/%(namelower)s"
+
+dependencies = [
+     ('FastQC', '0.10.1', '-Java-1.7.0_21'),
+     ('cutadapt', '1.5', '-goolf-1.4.10-Python-2.7.3'),
```

Comment:
if you include a dependency that uses a non-dummy toolchain, please make this easyconfig use that same toolchain, to make it clear